File: BTD Mod Helper Core/Api/UI/BlatantFavoritism.cs
using UnityEngine;

namespace BTD_Mod_Helper.Api
{
    internal class BlatantFavoritism
    {
        public static Color32 GetColor(string repoOwner)
        {
            switch (repoOwner)
            {
                case "doombubbles":
                    return new Color32(200, 0, 255, 255);
                case "gurrenm3":
                    return new Color32(255, 215, 0, 255);
                default:
                    return Color.white;
            }
        }
        
    }
}

File: Website/src/lib/blatant-favoritism.ts
export const getColor = (user: string) => {
  switch (user) {
    case "doombubbles":
      return "rgba(200, 0, 255, 255)";
    case "gurrenm3":
      return "rgba(200, 150, 255, 255)";
    default:
      return "white";
  }
};
